module/behaviour/planning/SimpleWalkPathPlanner/src/SimpleWalkPathPlanner.cpp

```diff
@@ -164,11 +164,11 @@ namespace module::behaviour::planning {
                     case message::behaviour::MotionCommand::Type::DIRECT_COMMAND: walkDirectly(); return;
 
                     case message::behaviour::MotionCommand::Type::BALL_APPROACH:
-                        determineSimpleWalkPath(ball, field, sensors, wantsTo, kickPlan, fieldDescription);
+                        determineSimpleWalkPath(&ball, &field, &sensors, &wantsTo, &kickPlan, &fieldDescription);
```

Comment:
I don't know what you're trying to do here but this is definitely wrong.
You're taking the address of all these things

Reply:
Yes, I'm still in the process of learning the ropes at the moment so to speak, I've been making commits to build using buildkite as since we're online my machine doesn't have the resources to build in a reasonable time and trying to force errors for troubleshooting. I've corrected it now so it builds and just fixing up formatting errors